app/pb_kits/playbook/pb_body/_body.jsx

```diff
@@ -40,11 +40,11 @@ const Body = (props: BodyProps) => {
   const ariaProps = buildAriaProps(aria)
   const dataProps = buildDataProps(data)
   const classes = classnames(
-    className,
     buildCss('pb_body_kit', color, status, {
       dark: dark,
     }),
-    spacing(props)
+    spacing(props),
+    className
```

Comment:
👌🏼 Thanks @jasoncypret! @powerhome/thunderbolts was about to open a PR for this.

Reply:
Cool! @drborges We are merging into Nitro today. Follow this PR to see when it lands: https://github.com/powerhome/nitro-web/pull/16180 CC